pages/api/app.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+function buildAppProfile(hostname, clientId) {
+  return {
+    "@context": "https://www.w3.org/ns/solid/oidc-context.jsonld",
+    client_id: clientId,
+    redirect_uris: [hostname],
+    client_name: "Inrupt PodBrowser",
+    grant_types: ["refresh_token", "authorization_code"],
+    response_types: ["code"],
```

Comment:
So ... per discussion with Aaron, the default values for grant_types and response_types should be correct.  So, whether you all want to specify grant_types and response_types, I'll leave it up to you all.  (I did ask Aaron if you all should.  When he gets back, I'll forward it on.  If he states otherwise, I'll let you all know).

Reply:
Per Aaron -- he said it's up to you all

> That's really up to them. There is a strong argument to explicitly defining the default values *in code*

Reply:
@ajacksified would you prefer this config to be removed and the defaults are taken?

Reply:
`    grant_types: ["refresh_token", "authorization_code"],
    response_types: ["code"],`
    
    ☝️  I have removed the above two config lines and tested that everything works as expected.